src/Command/Extension/SetExtensionVersionCommand.php

```diff
@@ -66,11 +70,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $versionReplacer = new VersionReplacer($version);
 
-        try {
-            $versionReplacer->setVersion($emConfFile, self::EMCONF_PATTERN);
-        } catch (\InvalidArgumentException $e) {
-            $io->error(sprintf('An error occurred while setting the ext_emconf.php version to %s.', $version));
-            return 1;
+        if (!$dryRun) {
```

Comment:
Could we just do an early return above L:71?  There is nothing to do below, in case this is a dry run. Would make the change a bit smaller.

Reply:
I've added a note here and below. If we like to keep this output, no.